lua/markdown.lua

```diff
@@ -153,20 +153,62 @@ function Math(el)
   return el
 end
 
+
+local function starts_with(str, start)
```

Comment:
a very useful function，could you add it to following file?

https://github.com/mokeyish/obsidian-enhancing-export/blob/master/lua/polyfill.lua

Reply:
will do